distributed/client.py

```diff
@@ -4007,6 +4007,36 @@ async def _get_task_stream(
         else:
             return msgs
 
+    async def _register_scheduler_plugin(self, plugin=None, **kwargs):
```

Comment:
Similar comment here 

```suggestion
    async def _register_scheduler_plugin(self, plugin, **kwargs):
```